src/main/java/use_case/execute_buy/ExecuteBuyInteractor.java

```diff
@@ -32,6 +32,8 @@ public ExecuteBuyInteractor(ExecuteBuyDataAccessInterface dataAccess, ExecuteBuy
      */
     @Override
     public void execute(ExecuteBuyInputData data) {
+        // TODO: commission fee?
```

Comment:
Try to make the TODOs complete sentences that clearly description what is the task to be done. I have encountered a few TODOs that are completely unclear in the codebase. It would be better if we can submit these as issues on GitHub.

Reply:
Corrine is responsible for this task, anyway I'll check all the todos for next commit.